addons/advanced_throwing/stringtable.xml

```diff
@@ -66,7 +66,7 @@
             <Russian>Включает визуализацию траектории броска (как полетит граната).</Russian>
             <Japanese>投てき軌道 (投てき物がどこに飛ぶか) の表示を有効化します。</Japanese>
             <Polish>Wyświetla wizualizację trasy przelotu granatu.</Polish>
-            <German>Aktiviert die Visualisierung des Wurfbogens (wohin das Objekt geworfen werden wird).</German>
+            <German>Aktiviert die Anzeige des Wurfbogens (wohin das Objekt geworfen wird).</German>
```

Comment:
"Aktiviert die Anzeige der erwarteten Flugkurve des Objekts" ? :D 
The "expected" part is kinda important. Because nothing guarantees that it will fly exactly that curve.
I preferred the previous variant.

Reply:
Aktiviert die Visualisierung der Flugbahn. (wohin das Objekt geworfen wird)